.github/workflows/build.yml

```
@@ -19,6 +19,8 @@ jobs:
         with:
           args: 
              "${{ steps.imagemagick.outputs.version }}"
+      - name: Test
+        run: "! rpm -qp --requires rpms/ImageMagick-libs-${{ steps.imagemagick.outputs.version }}.x86_64.rpm |grep -qL 'libcdt|libcgraph|libgvc|libgs|libMagickCore|libMagickWand'"
```

I'm not a big fan of using `!` even more when we are using pipes. That will for sure lead to unexpected behaviors and is very miss-leading.
```suggestion
        run: rpm -qp --requires rpms/ImageMagick-libs-${{ steps.imagemagick.outputs.version }}.x86_64.rpm | grep -qEv 'libcdt|libcgraph|libgvc|libgs|libMagickCore|libMagickWand'
```